demo/tcf/utils/borosMock.js

```diff
@@ -13,7 +13,7 @@ const mockedEmptyUserConsent = {
 }
 const mockedFullUserConsent = {
   isNew: false,
-  specialFeatureOptins: [],
```

Comment:
is this OK?
the TCData key is `specialFeatureOptins` (and is not an array)

Reply:
This is not the TCData is our domain object